sybil/integration/unittest.py

```diff
@@ -1,22 +1,28 @@
-from typing import TYPE_CHECKING
+from __future__ import annotations
```

Comment:
I don't like this, please don't use it.

Reply:
I have changed this to use quoted annotations rather than `from __future__ import annotations`.